pkg/scheduler/framework/runtime/framework.go

```diff
@@ -372,6 +372,19 @@ func NewFramework(r Registry, profile *config.KubeSchedulerProfile, stopCh <-cha
 		}
 	}
 
+	// Cache metric streams for prescore and score plugins.
```

Comment:
Can we have a separate function for L361 - L387?
If we add more extension points support on PluginEvaluationTotal in the future, here will get longer and longer.

Reply:
Hmmm, I separated a function "setInstrumentedPlugins" to do this, I can't think of a more cleaner way🤔